packages/electrode-archetype-react-app-dev/package.json

```diff
@@ -175,7 +173,8 @@
       "electrode-archetype-opt-karma": "../electrode-archetype-opt-karma",
       "electrode-archetype-opt-pwa": "../electrode-archetype-opt-pwa",
       "electrode-archetype-opt-react": "../electrode-archetype-opt-react",
-      "electrode-archetype-opt-sass": "../electrode-archetype-opt-sass"
+      "electrode-archetype-opt-sass": "../electrode-archetype-opt-sass",
+      "electrode-archetype-opt-critical-css": "../electrode-archetype-opt-critical-css"
```

Comment:
this is only for fyn's local install.  You need to add it to `optionalDependencies` above also.